File: InverDigit.c
/******************************************************************************

Invert every two integer from last....

Input:
7639

Output:
6793


Input:
73639

Output:
76393


*******************************************************************************/

#include <stdio.h>

int rev(int a){
    int s=0;
    while(a>0){
        s=(s*10)+(a%10);
        a/=10;
    }
    return s;
}

int main()
{
    int n;
    scanf("%d",&n);
    int m=0,t;
    while(n>0){
        t=n%10;
        n/=10;
        if(n==0){ 
            m=((m*10)+t);
            break;
        }
        m=((m*10)+n%10);
        m=((m*10)+t);
        n/=10;
    }
    printf("%d",rev(m));
    return 0;
}

File: InverDigit.java
/******************************************************************************

Invert every two integer from last....

Input:
7639

Output:
6793


Input:
73639

Output:
76393

*******************************************************************************/

import java.util.*;

public class Main
{
	public static void main(String[] args) {
	Scanner x=new Scanner(System.in);
	int n=x.nextInt();
	int r=0,f,l;
	while(n>0){
	    f=n%10;
	    n/=10;
	    if(n==0){
	        r=(r*10)+f;
	        break;
	    }
	    l=n%10;
	    r=(r*10)+l;
	    r=(r*10)+f;
	    n/=10;
	}
	while(r>0){
	    n=(n*10)+(r%10);
	    r/=10;
	}
	System.out.println(n);
	}
}
